.travis.yml

```diff
@@ -14,6 +14,10 @@ matrix:
       env: TOXENV=py36
     - python: 3.6
       env: TOXENV=docs
+    - python: 3.7
+      env: TOXENV=py37
+      dist: xenial
+      sudo: required
```

Comment:
Xenial now officially supports Python 3.7. You can drop the sudo.

https://blog.travis-ci.com/2018-11-08-xenial-release